Source/WebKit/WebProcess/WebPage/dmabuf/AcceleratedSurfaceDMABuf.h

```diff
@@ -41,10 +41,17 @@
 #include <WebCore/DRMDeviceNode.h>
 #include <atomic>
 #include <wtf/Lock.h>
-typedef void *EGLImage;
 struct gbm_bo;
 #endif
 
+#if OS(ANDROID)
+typedef struct AHardwareBuffer AHardwareBuffer;
+#endif // OS(ANDROID)
```

Comment:
The other #endif don't have the trailing comment, I think you can omit it.

Reply:
I think for the final version of the patch I'll remove the trailing `// OS(ANDROID)` in those cases where the corresponding `#if` is close by and it's obvious what the `#endif` is closing. Other ones, where the preprocessor directives are far from each other, are quite useful.